testsuite/MDAnalysisTests/data/coordinates/create_h5md_data.py

```diff
@@ -0,0 +1,90 @@
+import MDAnalysis as mda
+from MDAnalysis.tests.datafiles import COORDINATES_TRR, COORDINATES_TOPOLOGY
+try:
+    import pyh5md
+except ImportError:
+    HAS_PYH5MD = False
+else:
+    HAS_PYH5MD = True
```

Comment:
For this script, a developer will just have to install pyh5md. No need to check, just add a comment about where to find it and how to install
```suggestion
# To create the test trajectory, install pyh5md from https://github.com/pdebuyl/pyh5md
# with 'pip install pyh5md'
import pyh5md
```

It's ok if this fails with `ImportError` as it will rarely be run again.